blueman/gui/manager/ManagerDeviceList.py

```diff
@@ -407,20 +410,23 @@ def on_finished(fader: CellFade) -> None:
 
                 faderhandler = fader.connect("animation-finished", on_finished)
 
+            w = 14 * self.get_scale_factor()
+            h = 48 * self.get_scale_factor()
+
             to_store = {}
             if round(row["rssi"], -1) != round(rssi_perc, -1):
-                icon_name = "blueman-rssi-%d.png" % round(rssi_perc, -1)
-                icon = GdkPixbuf.Pixbuf.new_from_file(os.path.join(PIXMAP_PATH, icon_name))
+                icon_name = "blueman-rssi-%d.png" % round(rssi_perc)
```

Comment:
The -1 seems to be missing here.

Reply:
Ooh, good someone is paying attention 😁. I'll fix it later today.